### PR TITLE
JDBC Tasks of Marian Zlatev Tasks 6

### PR DESCRIPTION
**Pull request by NewestUser**
Storing one million rows in a minute and a half.

### CODE REVIEW

File: src/com/clouway/jdbc/BillionaireRegister.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  public void register(int dollars, int euro, int bgn) {
+    try {
+      PreparedStatement stmt = conn.prepareStatement("INSERT INTO billionaires (dollars, euro, bgn) VALUES(?,?,?)");
+      stmt.setInt(1, dollars);
+      stmt.setInt(2, euro);
+      stmt.setInt(3, bgn);
+      stmt.execute();
+      stmt.close();
```
Comment (mgenov) [inline src/com/clouway/jdbc/BillionaireRegister.java]: should be executed in finally to ensure that the prepared statement will be closed.

File: tests/com/clouway/jdbc/BillionaireRegisterTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  public void setUp() throws Exception {
+    conn = DriverManager.getConnection("jdbc:mysql://localhost:3306/examples", "root", "password");
+    random = new Random();
+    clearTableContent(conn);
+    conn.setAutoCommit(false);
+    register = new BillionaireRegister(conn);
+  }
+
+  @Test
+  public void add1MillionRows() throws Exception {
```
Comment (mgenov) [inline tests/com/clouway/jdbc/BillionaireRegisterTest.java]: How much it takes when you try to query on some of the columns ?

Comment (NewestUser) [inline tests/com/clouway/jdbc/BillionaireRegisterTest.java]: Queries on columns are fast. About 0.1 - 0.4 seconds. Even select * from billionaires is completed in less then 0.5 seconds

**Comment by mgenov** [comment]
LGTM !

**Pull request merged**